test/browser/connection.test.js

```diff
@@ -401,6 +401,38 @@ define(['shared_helper', 'chai'], function (helper, chai) {
         }
         closeAndFinish(done, realtime);
       });
+
+      /*
+       * Check behaviour when going through a proxy which doesn't send any xhr streaming
+       * responses until the stream closes (simulated by a transportparam interpreted by
+       * realtime)
+       */
+      it('connection behaviour with a proxy through which streaming is broken', function (done) {
```

Comment:
Also, I'm not comfortable adding a test which will break CI until that realtime change is on sandbox. I think the best option I can think of is to add this as an `it.skip` for now with a TODO/gh issue

Reply:
I've merged https://github.com/ably/realtime/pull/4231, will automatically deploy to sandbox in the next hour or so